airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/write/standard_insert/BigqueryBatchStandardInsertLoader.kt

```diff
@@ -162,14 +173,54 @@ class BigqueryBatchStandardInsertsLoaderFactory(
                 .setProject(bigquery.options.projectId)
                 .build()
 
+        val formatter: RecordFormatter =
+            when (dataChannelFormat) {
+                DataChannelFormat.PROTOBUF -> {
+                    if (config.legacyRawTablesOnly) {
+                        throw ConfigErrorException("Raw tables not supported in speed mode")
```

Comment:
checking: does platform fall back to normal mode for these connections?

Reply:
Nope! We will start failing the sync. But based on our discussion I was under the impression that we should not longer worry about raw tables right?

Reply:
Ughhhh! Alright let me implement the raw tables! :/

Reply:
done